libs/modelviewer/package.json

```diff
@@ -0,0 +1,35 @@
+{
+  "name": "@cc-components/modelviewer",
+  "version": "0.0.1",
+  "type": "module",
+  "main": "./dist/src/index.js",
+  "module": "./dist/src/index.js",
+  "types": "./dist/src/index.d.ts",
+  "scripts": {
+    "build": "tsc -b -f",
+    "test": "vitest",
+    "test:silent": "vitest --silent --run",
+    "test:coverage": "vitest run --coverage"
+  },
+  "dependencies": {
+    "@equinor/echo-3d-viewer": "^0.0.1",
+    "@cognite/reveal": "4.2.2",
+    "@cognite/sdk": "8.2.0",
+    "three": "0.154.0",
+    "rxjs": "^7.5.7"
+  },
+  "devDependencies": {
+    "@equinor/fusion-framework-module": "^4.2.5",
+    "@equinor/fusion-framework-module-http": "^5.1.1",
+    "@equinor/fusion-framework-react-app": "^4.1.8",
+    "react": "18.2.0",
+    "@types/three": "~0.154.0"
+  },
+  "peerDependencies": {
```

Comment:
Why peerdeps on a package that isnt going to be published?

Reply:
Using peer dependencies in a package that you're not planning to publish might seem counterintuitive at first, but there are several reasons why this might be done:

- Shared Dependencies: If you're working on a large project with multiple local packages that rely on the same dependency, you might use peer dependencies to ensure that all your packages use the same version of that shared dependency. This can help in keeping the overall bundle size smaller and avoid version conflicts.

- Development Consistency: In a team development environment, using peer dependencies for local packages ensures that all developers are working against the same version of a dependency, which can reduce bugs that arise from inconsistent environments.

- Interoperability: If your package is designed to work as a plugin or an add-on for another tool or library, defining a peer dependency can make sure that it remains compatible with the host tool or library, even if it's just for internal use within your organization.

- Documentation: Declaring a peer dependency is also a form of documentation. It makes it explicit that the package is intended to be used with certain versions of other packages, which can be helpful for new developers or for future reference.

- Workspace Management: In monorepos using peer dependencies can help manage and link local packages without publishing them to a package registry. This can simplify dependency resolution and improve build performance.